spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/couchbase/CouchbaseConfiguration.java

```diff
@@ -44,97 +36,27 @@
 
 	private final CouchbaseProperties properties;
 
-	public CouchbaseConfiguration(CouchbaseProperties properties) {
```

Comment:
This, along with other changes to `public` and `protected` methods on this class, are breaking API changes that we'd prefer to avoid if at all possible. If it's not possible, we'll need to see if we can provide a migration path via deprecation.